Assets/MixedRealityToolkit/Inspectors/Icons/MRTK_Loading_RotatingObject_Icon.gif.meta

```diff
@@ -0,0 +1,88 @@
+fileFormatVersion: 2
+guid: 5dba477ba83eb0b479990c3adad36d38
```

Comment:
This is reminder for me to go back and remove unnecessary icons that have been replaced by @cre8ivepark 's better icons